corehq/apps/api/tasks.py

```diff
@@ -8,5 +8,5 @@
 
 @periodic_task(run_every=crontab(minute=0, hour=0), queue='background_queue')
 def clean_api_access():
-    accessed = int(time.time()) - 30 * 24 * 3600  # only keep last 30 days
+    accessed = int(time.time()) - 90 * 24 * 3600  # only keep last 30 days
```

Comment:
The comment should also be updated.

also super weird tastypie uses an int for this time. it may be more readable with something similar to  `accessed = (datetime.utcnow() - timedelta(days=90) - datetime.datetime.utcfromtimestamp(0)).total_seconds()`

Reply:
Comment updated.

Not going to change the calculation since I find the current version easiest to read (subtract seconds from current epoch time).